src/stores/__tests__/errorStore.test.ts

```diff
@@ -0,0 +1,83 @@
+import IError from '~/models/IError';
+import httpStatusDescriptionCodeList from '~/codeLists/httpStatusDescriptionCodeList';
+import { ErrorStore } from '../errorStore';
+
+describe('errorStore.addError', () => {
+    it('Adds one error to error store', () => {
+        const errorStore = new ErrorStore;
+        const errorMessage = 'This is a test error';
+
+        errorStore.addError(errorMessage);
+
+        expect(errorStore.errorCount).toEqual(1);
+        expect(errorStore.latestError).toEqual(errorMessage);
+    });
+
+    it('Adds one error to error store, then removes it', () => {
+        const errorStore = new ErrorStore;
+        const errorMessage = 'This is a test error';
+
+        errorStore.addError(errorMessage);
+
+        expect(errorStore.errorCount).toEqual(1);
+
+        const error = errorStore.pop();
+
+        expect(error).toEqual(errorMessage);
+        expect(errorStore.errorCount).toEqual(0);
+    });
+
+    it('Adds one error with exception to error store', () => {
```

Comment:
I would change name of this test as "Adds one error to error store" if error is made as required param

Reply:
I would not make error a mandatory property, we currently have use cases where we show error messages, where it would be unneccessary to add an exception:
https://github.com/HSLdevcom/jore-map-ui/blob/20c626455a28815a44765c6a97e11a1cb27c6656/src/services/routePathLinkService.ts#L79